### PR TITLE
Minor Edit to sahih-bukhari.com.xml

### PR DESCRIPTION
**Pull request by Giltyhub**

### CODE REVIEW

File: src/chrome/content/rules/Sahih-Bukhari.com.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
-<ruleset name="sahih-bukhari.com">
+<ruleset name="Sahih-Bukhari.com">
 	<target host="sahih-bukhari.com" />
 	<target host="www.sahih-bukhari.com" />
 
+	<securecookie host=".+" name=".+" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Sahih-Bukhari.com.xml]: Generally, I would not add `<securecookie>` to existing rulesets, or the new ones, unless to fix an issue with the website.

**Comment by pipboy96** [comment]
Good overall, also off-topic but do you know Arabic language so you can test if ruleset causes issues with websites that can only be detected on some pages or only when logged in?

**Comment by pipboy96** [comment]
@J0WI :tada:

**Comment by Giltyhub** [comment]
@pipboy96 sure, if you want me to test any website in that language please hit me up (preferably in one of my PRs since i tend to not look at notifications/my email)

**Pull request merged**